tests/e2e_test.go

```diff
@@ -146,7 +138,7 @@ var _ = ginkgo.BeforeSuite(func() {
 
 func cleanup() {
 	if decider != nil {
-		cancelDecider()
+		cancelFn()
```

Comment:
We should lift this `cancelFn` out of this `if` block. It can be called whether or not we have a decider running.

Reply:
Addressed in 3d7e3c0